src/components/app/App.js

```diff
@@ -1,24 +1,65 @@
 import React from 'react';
-import logo from '../../logo.svg';
-import './App.css';
+
+import { Header } from '../header/HeaderFromLecture';
+import { Footer } from '../footer/Footer';
+import  { PostCard as Card } from '../post-card/PostCard';
+import { usersList, postsList } from '../../constants';
+import { allComments } from '../../constants';
+import {UserCard} from '../user-card/UserCard' 
+
+import './App.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
 function App() {
+
+  // const renderList = () => {
+  //   const res = [];
+
+  //   for (let i = 0; i < 6; i++) {
+  //     const item = postsList[i];
+  //     res.push(<Card post={item} key={item.id}/>)
+  //   }
+
+  //   return res;
+  // };
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Header />
+      <div>content will appear here soon</div>
+
+      {/* <div className="d-flex posts-container">
+       {
+          renderList()
+       }
+      </div> */}
+
+      <div className="d-flex posts-container">
+        {
+          postsList.map((item, index) => {
+            const odd = index % 2 !== 0;
+            const comment = [];
+            allComments.filter ( value => {
+             if (value.post_id === item.id) {
+             comment.push (value)  
```

Comment:
You don't need to use extra array to saving result
function `filter` return an array
so basically you can just assign to variable the result of calling `filter`
and it would be
````

const comments = allComments.filter(value => value.post_id === item.id);
````